sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -373,7 +373,8 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
         namedGroupingExpressions, aggregateExpressions, rewrittenResultExpressions, child) =>
 
         if (aggregateExpressions.exists(PythonUDF.isGroupedAggPandasUDF)) {
-          throw QueryCompilationErrors.groupAggPandasUDFUnsupportedByStreamingAggError()
+          throw new AnalysisException(
+            "Streaming aggregation doesn't support group aggregate pandas UDF")
```

Comment:
Could you clarify why did you make this replacement?

Reply:
Because the existing `groupAggPandasUDFUnsupportedByStreamingAggError` is not user-facing error, so I think we might not want to reveal this to the error class.

Also refer to https://github.com/apache/spark/pull/35656/files#r814517495.

Maybe do we also want to leave some comment to the code base?